.github/actions/run-tests/action.yml

```diff
@@ -24,114 +24,126 @@ inputs:
 
-  - name: set job name
-    shell: bash
-    id: job
-    run: echo "::set-output name=name::testing"
+    - name: run tests
+      shell: bash
+      run: |
+        cat <<EOF | kubectl apply -f -
+        apiVersion: batch/v1
+        kind: Job
+        metadata:
+          name: "${{ steps.job.outputs.name }}"
+        spec:
+          template:
+            metadata:
+              annotations:
+                cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
+            spec:
+              containers:
+              - name: tests
+                image: ${{ inputs.testing_docker_image }}
+                resources:
+                  requests:
+                    memory: "512Mi"
+                    cpu: "250m"
+                  limits:
+                    memory: "512Mi"
+                    cpu: "250m"
+                command:
+                  - /bin/bash
+                  - -c
+                  - |
+                    HOSTS="${{ inputs.dns_records_to_resolve }}"
+                    for HOST in \$HOSTS; do
+                      HOST_ENTRY=""
+                      until [[ ! -z "\$HOST_ENTRY" ]]; do sleep 1; HOST_ENTRY=\$(getent hosts \$HOST); done;
+                      echo "\$HOST_ENTRY" >> /etc/hosts
+                      echo "DNS record for \$HOST resolved successfully!"
+                    done
 
-  - name: create configuration.py secret
-    shell: bash
-    run: |
-      cat << EOF > configuration.py
-      ${{ inputs.configuration }}
-      EOF
-      kubectl create secret generic configuration --from-file=configuration=./configuration.py
-     
-  - name: run tests
-    shell: bash
-    run: |
-      cat <<EOF | kubectl apply -f -
-      apiVersion: batch/v1
-      kind: Job
-      metadata:
-        name: "${{ steps.job.outputs.name }}"
-      spec:
-        template:
-          metadata:
-            annotations:
-              cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
-          spec:
-            containers:
-            - name: tests
-              image: ${{ inputs.testing_docker_image }}
-              resources:
-                requests:
-                  memory: "512Mi"
-                  cpu: "250m"
-                limits:
-                  memory: "512Mi"
-                  cpu: "250m"
-              command:
-                - /bin/bash
-                - -c
-                - |
-                  cd tests
-                  pytest -m "${{ inputs.marker_expression }}" -s -vvv --testbed=${{ inputs.testbed }} --alluredir=/tmp/allure-results ${{ inputs.additional_args }}
-                  ret=\$?
-                  # sleep some time to be able to download the Allure results
-                  sleep 60
-                  exit \$ret
-              volumeMounts:
+                    cd tests
+                    pytest -m "${{ inputs.marker_expression }}" -s -vvv --testbed=${{ inputs.testbed }} --alluredir=/tmp/allure-results ${{ inputs.additional_args }}
```

Comment:
Is it automatically ignored if `inputs.additional_args` is not defined since it is a non-required input? Or do we need something like `${{ inputs.additional_args || '' }}`?

Reply:
By default it's just going to be an empty string adding the fallback isn't necessary.